packages/fe-fpm-writer/templates/filter/Controller.ts

```diff
@@ -0,0 +1,24 @@
+import Filter from 'sap/ui/model/Filter';
+import FilterOperator from 'sap/ui/model/FilterOperator';
+
+/**
+ * Custom filter
+ * @param sValue selected filter item
+ * @returns new Filter
+ */
+export function <%- (typeof eventHandlerFnName !== 'undefined' && eventHandlerFnName) || 'itemsFilter' %>(sValue: string) {
```

Comment:
Same as before: Don't you set the value all the time in the `applyEventHandlerConfiguration` function? If yes, please just use `eventHandlerFnName` and don't do the additional checks and fallbacks.
(the same for the `Controller.js` file)

Reply:
Same as above